src/test-helpers/e2e-helpers/mobile/execute.ts

```diff
@@ -0,0 +1,11 @@
+import { Browser } from 'webdriverio'
+
+/** Execute a function in browser context. */
+const execute = (browser: Browser<any>, func: (...args: any) => any, ...args: any[]) => {
+  // https://github.com/webdriverio/webdriverio/issues/6206
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  return browser.execute(func, ...args)
+}
```

Comment:
This function just wraps `browser.execute`. Can you call `browser.execute` directly instead?

Reply:
I did this because `browser.execute` has some type problems and I didn't want to add lines 5-6-7 to everywhere that call `browser.execute`.

Reply:
I think I fixed it in 556c93b28

Reply:
😄 Thank you!